appservice/src/intTest/groovy/nu/yona/server/SystemMessagesTest.groovy

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server
+
+import groovy.json.*
+
+class SystemMessagesTest extends AbstractAppServiceIntegrationTest
```

Comment:
This name should be singular (``SystemMessageTest``), because it test the concept of a system message.